snuba/consumers/consumer.py

```diff
@@ -521,14 +521,12 @@ class MultistorageKafkaPayload(NamedTuple):
 
 def process_message(
     processor: MessageProcessor,
-    consumer_group: str,
     snuba_logical_topic: SnubaTopic,
     message: Message[KafkaPayload],
 ) -> Union[None, BytesInsertBatch, ReplacementBatch]:
     local_metrics = MetricsWrapper(
         metrics,
         tags={
-            "consumer_group": consumer_group,
```

Comment:
so we don't need the consumer group in the metrics anymore?

Reply:
I don't think it's all that useful here, it just tells us about invalid messages on the source topic which will be the same regardless of consumer group. It's not worth passing it in here for.

Reply:
I had to add it so that the global dashboard filters work, it's actually there not to break down by consumer group but to filter approximately by dataset (since it's the same metric across datasets)

`snuba_logical_topic` can go, on the other hand. It's not used in dashboards at all, and introducing it would be cumbersome.